extra_data/export.py

```diff
@@ -258,6 +295,13 @@ def main(argv=None):
         "--key", help="Stream only matching keys ('*' is a wildcard)",
         default='*',
     )
+    ap.add_argument(
+        "--append-detector-modules", help="combine multiple module sources \
+           into one (will only work for AGIPD data currently).",
```

Comment:
```suggestion
        "--append-detector-modules", help="combine multiple module sources "
        "into one (will only work for AGIPD data currently).",
```

This is a neater way of splitting strings over multiple lines (it avoids leaving a big gap in the resulting string).

Reply:
Yes, that's better indeed